cmd/support-top-net.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2015-2022 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"context"
+	"time"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/minio/cli"
+	"github.com/minio/madmin-go/v3"
+	"github.com/minio/mc/pkg/probe"
+)
+
+var supportTopNetFlags = []cli.Flag{
+	cli.IntFlag{
+		Name:  "count, c",
+		Usage: "show up to N Nets",
+		Value: 10,
+	},
```

Comment:
Is this copied from somewhere?

```suggestion
	cli.IntFlag{
		Name:  "count, c",
		Usage: "show top count interfaces",
		Value: 10,
	},
```
How is this determined?